types/chrome/test/index.ts

```diff
@@ -848,6 +848,13 @@ function testStorage() {
         var myNewValue: { x: number } = changes["myKey"].newValue;
         var myOldValue: { x: number } = changes["myKey"].oldValue;
     });
+
+    chrome.storage.sync.getKeys(); // $ExpectType Promise<string[]>
+    chrome.storage.sync.getKeys((keys) => { // $ExpectType void
+      keys;  // $ExpectType string[]
```

Comment:
There is just a problem with the indentation of my suggestion 😅
```suggestion
        keys;  // $ExpectType string[]
```